uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/modules/apis.js

```diff
@@ -0,0 +1,10 @@
+define([
+    'common/angular',
+    'common/modules/apis',
+], function (angular) {
+    'use strict';
+
+    return angular.module('leave-absences.apis', [
+        'common.apis',
```

Comment:
‘common.apis’ is needed as its extending the api services defined in common, if not included then injector is not able to locate entitlementAPIprovider and gives this error 
`
Error: [$injector:unpr] http://errors.angularjs.org/1.5.9/$injector/unpr?p0=apiProvider%20%3C-%20api%20%3C-%20EntitlementAPI%20%3C-%20Entitlement
`